src/main/java/org/broadinstitute/hellbender/utils/iterators/AlignmentContextIteratorFactory.java

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.hellbender.utils.iterators;
```

Comment:
I will make this live in the `org.broadinstitute.hellbender.utils.locusiterator` package.